torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -295,17 +354,23 @@ class TensorExprFuser {
     // Now all the nodes that we're going to fuse are moved next to the fusion
     // group, so we can safely merge them into the fusion group subgraph.
     fusion_group = getOrCreateTensorExprSubgraph(fusion_group);
+
+    std::unordered_map<Value*, Value*> vmap;
     for (auto n : nodes_to_merge) {
       GRAPH_UPDATE("Merging ", getHeader(n));
-      SubgraphUtils::mergeNodeIntoSubgraph(n, fusion_group);
+      SubgraphUtils::mergeNodeIntoSubgraph(n, fusion_group, vmap);
```

Comment:
can we wrap mergeNodeIntoSubgraph and updateTypeinfoMapWithVmap into one single helper? Otherwise, there's always a possibility somewhat will insert another graph transformation and now we are running into the same issue of having the same address as a key and a value.

Reply:
What does this mean?
> there's always a possibility somewhat will insert another graph transformation